css/wandering-cubes.css

```diff
@@ -22,9 +22,9 @@
 }
 
 @-webkit-keyframes sk-cubemove {
-  25% { -webkit-transform: translateX(22px) rotate(-90deg) scale(0.5) }
-  50% { -webkit-transform: translateX(22px) translateY(22px) rotate(-180deg) }
-  75% { -webkit-transform: translateX(0px) translateY(22px) rotate(-270deg) scale(0.5) }
+  25% { -webkit-transform: translateX(42px) rotate(-90deg) scale(0.5) }
+  50% { -webkit-transform: translateX(42px) translateY(42px) rotate(-180deg) }
+  75% { -webkit-transform: translateX(0px) translateY(42px) rotate(-270deg) scale(0.5) }
```

Comment:
Not sure why these weren't on the same 42px scale and the non-vendor-prefixed keyframes, so I adjusted them. If there was a reason, I'm happy to revert.